src/components/Send/SendScreen.js

```diff
@@ -750,12 +754,14 @@ class SendScreen extends Component<Props, State> {
             assetsMetadata={availableAssets}
             unselectEnabled={false}
           />
+          {this.state.fee == null &&
```

Comment:
@v-almonacid wouldn't the fee change if we change the amount and it requires more utxos?

Reply:
(meaning that we need to display the indicator again)